pkg/runtime/grpc/oci.go

```diff
@@ -123,6 +123,11 @@ func (r *Runtime) runGadgetOnTargets(
 	}
 
 	wg.Wait()
+	// If we are in interactive mode, stop local operators after all remote targets
+	// have stopped their operators and "returned"
+	if !gadgetCtx.UseInstance() {
+		gadgetCtx.StopLocalOperators()
+	}
```

Comment:
Why not to call it in all cases?

Reply:
Now that you asked that, it might make sense to just run it everytime.

I though in the daemon or headless variant we don't want to stop the local operators - but they are only on the client anyway...
Maybe I missed something now, but is something like the `sort` or `combiner` operator running in a detached mode. No, right?

Reply:
Ah, good point. We're running those operators even in detached mode. I think we need to run them to get their parameters, but since these are operators on the client, probably those parameters should only be set on `attach`.

@flyth any thoughts on this?

Reply:
> Ah, good point. We're running those operators even in detached mode. I think we need to run them to get their parameters, but since these are operators on the client, probably those parameters should only be set on attach.

We could also keep the behavior the same as before this PR. We always stopped the local operators when the context is done

Reply:
I agree, feel free then to remove the `if gadgetCtx.UseInstance() {` conditional and merge if you think it's ready to go.

Reply:
Done